cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -45,14 +45,12 @@ pub async fn create_swap<D, A: ActorState>(
     // construct a generator that watches alpha and beta ledger concurrently
     let mut generator = Gen::new({
         let dependencies = dependencies.clone();
-        |co| {
-            async move {
-                future::try_join(
-                    watch_alpha_ledger(&dependencies, &co, swap.alpha_htlc_params(), at),
-                    watch_beta_ledger(&dependencies, &co, swap.beta_htlc_params(), at),
-                )
-                .await
-            }
+        |co| async move {
```

Comment:
This block keeps getting re-formatted in various PRs.  Seems some tools are arguing about whether the `async move` needs to be inside another block or not.